.github/workflows/assets.yml

```diff
@@ -79,10 +79,13 @@ jobs:
              mv assets/kernel assets/kernel.gz
              gzip -d assets/kernel.gz
           fi
-      - name: Build SBOM and collected sources
+      - name: Pull eve-sources and publish collected_sources.tar.gz to assets
         run: |
-          make sbom collected_sources compare_sbom_collected_sources
-          mv $(make sbom_info collected_sources_info) assets/
+          EVE_SOURCES=lfedge/eve-sources:${{ env.TAG }}-${HV}-${{ env.ARCH }}
+          docker pull "$EVE_SOURCES"
+          docker create --name eve_sources $EVE_SOURCES" bash
```

Comment:
Looks like unbalanced quotes on this line

Reply:
fixed